sourcegraph/sourcegraph
# app: Add app outbound url params on notebooks page

In Sourcegraph App, add analytics URL parameters to the link URLs in the Notebooks page that go to example Notebooks on Dotcom.

The outbound url params are added using `addSourcegraphAppOutboundUrlParameters()`

The URL params are added only in Sourcegraph App mode.

## Test plan

- Start Sourcegraph in App mode `sg start app`
- Visit the notebooks getting started page
- Check links have outbound URL params added
- Start Sourcegraph in non-App mode
- Check links have the original behavior

## App preview:

- [Web](https://sg-web-marek-app-outbound-params.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
